SwiftTweaks/TweakViewData.swift

```diff
@@ -16,6 +16,7 @@ internal enum TweakViewData {
 	case doubleTweak(value: Double, defaultValue: Double, min: Double?, max: Double?, stepSize: Double?)
 	case color(value: UIColor, defaultValue: UIColor)
 	case stringList(value: StringOption, defaultValue: StringOption, options: [StringOption])
+	case action(value: TweakCallbacks)
```

Comment:
Similar to my comment about not using `TweakBinding`, it's kind of the way the system is designed right now - it requires a valid `TweakableType` `value` from `TweakViewData`. Here we could probably skip the associated value and run callbacks some other way, rather than getting it directly from `viewData`, but on the other hand, it is there so we might as well use it.
I think that also, at least in this context, `TweakCallbacks` is closer to `CGFloat` or `UIColor` which you do see as associated values few lines higher (in other words - a container for data - floating point number or a list of callbacks) than `TweakBindings`.